assignment.md

```diff
@@ -11,47 +11,102 @@ If the version control concepts or `git` commands are still confusing to you, co
 
-    > Your answer here
+    > git push upstream main 
+Username for 'https://github.com': Joody90
+Password for 'https://Joody90@github.com': 
+remote: Permission to WCM-datascibasics/version_control_assignment.git denied to Joody90.
+fatal: unable to access 'https://github.com/WCM-datascibasics/version_control_assignment/': The requested URL returned error: 403
+
+
+I don't have access to the main folder in the remote repository. 
 
 
  4. Now clone the repository for the [class website](https://github.com/WCM-datascibasics/wcm-datascibasics.github.io), and in the class website repository:
     - a. Explore the version history by visualizing it as a graph (remember the dog meme?).
         ```bash
+git log --all --decorate --oneline --graph
+* a3f66fd (HEAD -> master, origin/master, origin/HEAD) swapped 2 weeks
+* cd0d211 python notes repo link
+* df90802 updated syllabus with presidents day
+* e0029df place holder schedule
+* 7462142 update lecture link for week 3
+* 8fced37 assignment 2 updated
+* 2a2bfd5 version control chapter
+* 13e5722 minor improvements to instructions
+* dd4719c fixed assignment 1 link
+* 0392554 fixed syllabus
+* 48e689c updated syllabus with MLK day
+* e96eba0 added dave's slides link
+* fbb9e8d fixed typo
+* c152f3c added ch2 assignment
+* cd40b26 fixed typo in assignment 1
+* b3455f6 fixed missing link
+* 7cd4587 update links to reflect erlative links
+* fb12bef jekyll config additions for relative links
+* 7ed94e9 dash fix
+* d6f3ec1 fixed link error
+* 2c05357 update links
+* 9586a22 removed type app
+* 00e8ae5 readme and try new size
+* 793782a Set theme jekyll-theme-minimal
+* b7a7152 initial transfer to no annex repo
         # git command and output here
         ```
 
     - b. Who was the last person to modify `README.md`? (Hint: use `git log` with an argument)
         ```bash
+git log README.md
+commit 00e8ae51a08a0f3754ab228f62b51651595c4426
+Author: Xihe Xie <axiezai@gmail.com>
+Date:   Mon Dec 28 16:01:51 2020 -0500
         # git command and output here
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+ added ch2 assignment
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > the changes in that file are undone and saved into a special place.
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > f1a890c (refs/stash) WIP on master: a3f66fd swapped 2 weeks
```

Comment:
WIP = work in progress, the key being your "HEAD" or latest commit is present but with an additional indicator that you have saved WIP tracked by git.